com.github.rhyme_player.rhyme.appdata.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2021 Abdallah-Moh <abdallah.mam29@gmail.com> -->
+<component type="desktop">
```

Comment:
this wont validate: missing releases, project license, oars, screenshots. at least.